macosx/mac-app.py

```diff
@@ -49,7 +49,7 @@
 parser.add_argument('-s', '--script', \
   help = 'path of {0}\'s main script; you should use an absolute path, \
   so that the application bundle can be moved to another \
-  directory'.format(appinfo.appname), default = '{0}/{1}'.format(root, appinfo.name))
+  directory'.format(appinfo.appname), default = '{0}/{1}/__main__.py'.format(root, appinfo.name))
```

Comment:
In this PR you do not rename the `frescobaldi_app` directory, so this should be `default = '{0}/{1}_app/__main__.py'.format(root, appinfo.name)`.
I made this change locally and tested building the application bundle and it worked.